### PR TITLE
kata-deploy: stop mentioning qemu-virtiofsd, as the default qemu supports virtiofsd already (plus some cleanups)

### PR DESCRIPTION
**Pull request by fidencio**
This series is based atop of #1423, but I still don't feel comfortable to close that one in favour of this one here.

Fixes: #1424

### CODE REVIEW

**Comment by fidencio** [comment]
@egernst, @wainersm, may I bother you for a review here?

**Review by pmores — CHANGES_REQUESTED** [review]

File: docs/how-to/how-to-use-virtio-fs-with-kata.md [hunk 1]
```diff
@@ -1,61 +1,11 @@
...
-- [Install Kata Containers with virtio-fs support](#install-kata-containers-with-virtio-fs-support)
-- [Run a Kata Container utilizing virtio-fs](#run-a-kata-container-utilizing-virtio-fs)
 
 ## Introduction
 
 Container deployments utilize explicit or implicit file sharing between host filesystem and containers. From a trust perspective, avoiding a shared file-system between the trusted host and untrusted container is recommended. This is not always feasible. In Kata Containers, block-based volumes are preferred as they allow usage of either device pass through or `virtio-blk` for access within the virtual machine.
 
-As of the 1.7 release of Kata Containers, [9pfs](https://www.kernel.org/doc/Documentation/filesystems/9p.txt) is the default filesystem sharing mechanism. While this does allow for workload compatibility, it does so with degraded performance and potential for POSIX compliance limitations.
+As of the 2.0 release of Kata Containers, [virtio-fs](https://virtio-fs.gitlab.io/) is the default filesystem sharing mechanism. While this does allow for workload compatibility, it does so with degraded performance and potential for POSIX compliance limitations.
```
[the remark this answers is not in the document]
Comment (fidencio) [reply docs/how-to/how-to-use-virtio-fs-with-kata.md:9]: I don't think it applies, but I think the sentence should be rewritten.  Let me do it and re-submit the series.
Thanks for the review, @pmores!

[the remark this answers is not in the document]
Comment (fidencio) [reply docs/how-to/how-to-use-virtio-fs-with-kata.md]: I ended up just removing that bit, as it seems the best to do for now.

We may need to re-write the document later on, but I'll leave that for later on.
I'm marking this conversation as resolved.

**Review by pmores — APPROVED** [review]

**Review by lifupan — APPROVED** [review]

**Pull request merged**